README.md

```diff
@@ -4,10 +4,13 @@
 
 ### `solx` is an optimizing compiler for Solidity
 
-The project aims to bring state-of-the-art optimizing compilers to Ethereum by:
-- Reducing gas consumption of EVM code.
-- Eliminating the need for inline assembly.
-- Providing industry-grade tools like an interactive debugger.
+It passes [our test suite](https://github.com/matter-labs/era-compiler-tests), which includes tests from the `solc` project and real-world projects. All `solc` tests are updated with each new `solc` release. Arbitrary contracts are expected to compile correctly, except for stack-too-deep issues. Benchmarks indicate that `solx` generates larger code than `solc` but reduces gas consumption on average.
+
+#### Our mid-term goals:

```

Comment:
I would add that we're going to solve the bytecode size problem as well. At least by working on the `-Oz` mode.

Reply:
I didn't mention Oz, as for now I expect the most of optimizations we are to do will improve on both size and gas consumption. I expect we are ~1 year ahead of when it could start contradicting each other.